tests/frontend/test_kibana_app/cypress/integration/pageobjects/wazuh-menu/wazuh-menu.page.js

```diff
@@ -1,3 +1,74 @@
-export const wazuhButtonSelector = '.eui > .euiFlexGroup';
-export const menuListButtonsSelector = '[class="euiButtonEmpty euiButtonEmpty--text wz-menu-button "]';
-export const subMenuListButtonsSelector = '[class="euiSideNavItem euiSideNavItem--trunk"]';
+export const wazuhButton = '.eui > .euiFlexGroup';
+
+//region Menu
+export const modulesButton = '.wz-menu-sections > :nth-child(1) > .euiButtonContent';
+export const managementButton = '.wz-menu-sections > :nth-child(2) > .euiButtonContent';
+export const agentsButton = '.wz-menu-sections > :nth-child(3) > .euiButtonContent';
+export const toolsButton = '.wz-menu-sections > :nth-child(4) > .euiButtonContent';
+export const securityButton = '.wz-menu-sections > :nth-child(6) > .euiButtonContent';
+export const settingsButton = '.wz-menu-sections > :nth-child(7) > .euiButtonContent';
+//endregion Menu
+
+//region SubMenu
+
+//region Modules
+export const securityEventsButton = ':nth-child(1) > .euiSideNav > .euiSideNav__content > .euiSideNavItem--root > .euiSideNavItem__items > :nth-child(1) > .euiSideNavItemButton > .euiSideNavItemButton__content > .euiSideNavItemButton__label';
+export const integrityMonitoringButton = ':nth-child(1) > .euiSideNav > .euiSideNav__content > .euiSideNavItem--root > .euiSideNavItem__items > :nth-child(2) > .euiSideNavItemButton > .euiSideNavItemButton__content > .euiSideNavItemButton__label';
+export const amazonAwsButton = ':nth-child(1) > .euiSideNav > .euiSideNav__content > .euiSideNavItem--root > .euiSideNavItem__items > :nth-child(3) > .euiSideNavItemButton > .euiSideNavItemButton__content > .euiSideNavItemButton__label';
+export const googleCloudPlatformButton = ':nth-child(1) > .euiSideNav > .euiSideNav__content > .euiSideNavItem--root > .euiSideNavItem__items > :nth-child(4) > .euiSideNavItemButton > .euiSideNavItemButton__content > .euiSideNavItemButton__label';
```

Comment:
all these selectors will give us problems in the future... 
they are too attached to the page structure, like an absolut (ish) path

we need to modify the app to add data-cy attributes to these components.

Reply:
if it deserves it, let's create an issue in wazuh-kibana-app

Reply:
yes, exactly, we need to start adding more 'data-cy' attributes to be able to modify these selectors.